src/components/Card/LandingPageCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import propTypes from 'prop-types';
+const LandingPageCard = ({ image, title }) => {
+  return (
+    <>
+      <div className=" max-h-lg max-w-sm sm:w-1/2 lg:w-1/4  overflow-hidden shadow-lg transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-105 rounded">
+        <img className="w-full" src={image} alt="Train to Maintain landing" />
+        <div className="font-bold text-xl text-purple-600 text-center p-10">
```

Comment:
lets reduce the padding a bit and larger text
```suggestion
        <div className="font-bold text-3xl text-purple-600 text-center p-5">
```